src/main/scala/com/amazon/deequ/VerificationResult.scala

```diff
@@ -96,9 +96,10 @@ object VerificationResult {
       data: DataFrame): DataFrame = {
 
     val columnNamesToMetrics: Map[String, Column] = verificationResultToColumn(verificationResult)
+    val columnsAliased = columnNamesToMetrics.toSeq.map { case (name, col) => col.as(name) }
```

Comment:
Is the `toSeq` required? It seem to work just fine without it for me

Reply:
Removing `toSeq` works for you without any other changes? For me the `columnsAliased` definition is still fine without `toSeq` (it just becomes an `Iterable[Column]` instead of `Seq[Column]`), but then trying to prepend `col("*")` to the list of columns doesn't compile.

Reply:
Also, the dataframe select method for `Column`s is only defined for `Column*`, and vararg expansion is only defined for seqs as far as I can tell, so I don't think there's any getting around `toSeq` at some point in the process.